[2025-10-29T16:13:01.317+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T16:12:00+00:00 [queued]>
[2025-10-29T16:13:01.328+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T16:12:00+00:00 [queued]>
[2025-10-29T16:13:01.328+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-10-29T16:13:01.344+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): ConsumKafka> on 2025-10-29 16:12:00+00:00
[2025-10-29T16:13:01.353+0000] {standard_task_runner.py:60} INFO - Started process 737 to run task
[2025-10-29T16:13:01.360+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag1_calcul_trajet_kafka', 'ConsumKafka', 'scheduled__2025-10-29T16:12:00+00:00', '--job-id', '305', '--raw', '--subdir', 'DAGS_FOLDER/dag1_kafka_processing.py', '--cfg-path', '/tmp/tmptfdv882d']
[2025-10-29T16:13:01.362+0000] {standard_task_runner.py:88} INFO - Job 305: Subtask ConsumKafka
[2025-10-29T16:13:01.433+0000] {task_command.py:423} INFO - Running <TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T16:12:00+00:00 [running]> on host 46f6cfd4d166
[2025-10-29T16:13:01.528+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1_calcul_trajet_kafka' AIRFLOW_CTX_TASK_ID='ConsumKafka' AIRFLOW_CTX_EXECUTION_DATE='2025-10-29T16:12:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-29T16:12:00+00:00'
[2025-10-29T16:13:01.539+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.8', 9092)]>: connecting to kafka:9092 [('172.18.0.8', 9092) IPv4]
[2025-10-29T16:13:01.544+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <checking_api_versions_recv> [IPv4 ('172.18.0.8', 9092)]>: Broker version identified as 2.6
[2025-10-29T16:13:01.544+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Connection complete.
[2025-10-29T16:13:01.547+0000] {subscription_state.py:181} INFO - Updating subscribed topics to: ('source',)
[2025-10-29T16:13:01.548+0000] {dag1_kafka_processing.py:49} INFO - âœ… Consommateur Kafka crÃ©Ã© pour le topic 'source'. En attente de messages...
[2025-10-29T16:13:01.553+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.8', 9092)]>: connecting to kafka:9092 [('172.18.0.8', 9092) IPv4]
[2025-10-29T16:13:01.554+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Connection complete.
[2025-10-29T16:13:01.555+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Closing connection. 
[2025-10-29T16:13:01.658+0000] {cluster.py:401} INFO - Coordinator for group/***-consumers-dag1 is ('coordinator-1', 'kafka', 9092, None)
[2025-10-29T16:13:01.659+0000] {base.py:788} INFO - Discovered coordinator coordinator-1 for group ***-consumers-dag1
[2025-10-29T16:13:01.660+0000] {base.py:848} INFO - Starting new heartbeat thread
[2025-10-29T16:13:01.661+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group ***-consumers-dag1
[2025-10-29T16:13:01.661+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: NodeNotReadyError: coordinator-1
[2025-10-29T16:13:01.662+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.8', 9092)]>: connecting to kafka:9092 [('172.18.0.8', 9092) IPv4]
[2025-10-29T16:13:01.663+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Connection complete.
[2025-10-29T16:13:01.764+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-29T16:13:01.768+0000] {base.py:637} INFO - Received member id kafka-python-2.2.15-7108b072-5f18-4267-a0df-501b305f7964 for group ***-consumers-dag1; will retry join-group
[2025-10-29T16:13:01.771+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: [Error 79] MemberIdRequiredError
[2025-10-29T16:13:01.772+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-29T16:13:04.776+0000] {base.py:594} INFO - Successfully joined group ***-consumers-dag1 <Generation 86 (member_id: kafka-python-2.2.15-7108b072-5f18-4267-a0df-501b305f7964, protocol: range)>
[2025-10-29T16:13:04.776+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-10-29T16:13:04.780+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('source', 0)]
[2025-10-29T16:13:04.781+0000] {consumer.py:249} INFO - Setting newly assigned partitions (('source', 0),) for group ***-consumers-dag1
[2025-10-29T16:13:04.787+0000] {dag1_kafka_processing.py:60} INFO - ðŸ“© Message reÃ§u : {'confort': 'medium', 'prix_base_per_km': 2, 'properties-client': {'logitude': 2.391075974219345, 'latitude': 48.809377866725406, 'nomclient': 'FALL', 'telephoneClient': '060786575'}, 'properties-driver': {'logitude': 3.6925385718213466, 'latitude': 40.394816570797595, 'nomDriver': 'DIOP', 'telephoneDriver': '0760786575'}}
[2025-10-29T16:13:04.791+0000] {base.py:1052} INFO - Stopping heartbeat thread
[2025-10-29T16:13:04.792+0000] {base.py:891} INFO - Leaving consumer group (***-consumers-dag1).
[2025-10-29T16:13:04.800+0000] {base.py:906} INFO - LeaveGroup request for group ***-consumers-dag1 returned successfully
[2025-10-29T16:13:04.801+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Closing connection. 
[2025-10-29T16:13:04.802+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Closing connection. 
[2025-10-29T16:13:04.803+0000] {python.py:201} INFO - Done. Returned value was: {'confort': 'medium', 'prix_base_per_km': 2, 'properties-client': {'logitude': 2.391075974219345, 'latitude': 48.809377866725406, 'nomclient': 'FALL', 'telephoneClient': '060786575'}, 'properties-driver': {'logitude': 3.6925385718213466, 'latitude': 40.394816570797595, 'nomDriver': 'DIOP', 'telephoneDriver': '0760786575'}}
[2025-10-29T16:13:04.829+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag1_calcul_trajet_kafka, task_id=ConsumKafka, execution_date=20251029T161200, start_date=20251029T161301, end_date=20251029T161304
[2025-10-29T16:13:04.869+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-29T16:13:04.901+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
